titus-server-master/src/main/java/com/netflix/titus/master/jobmanager/service/service/action/CreateOrReplaceServiceTaskActions.java

```diff
@@ -88,12 +90,13 @@ private static TitusChangeAction createResubmittedTaskChangeAction(EntityHolder
 
     private static List<ModelActionHolder> createNewTaskModelAction(TitusChangeAction.Builder changeActionBuilder, ServiceJobTask newTask, Retryer newRetryer) {
         List<ModelActionHolder> actions = new ArrayList<>();
+        String summary = "Creating new task entity holder";
 
-        TitusModelAction.Builder modelBuilder = TitusModelAction.newModelUpdate(changeActionBuilder).summary("Creating new task entity holder");
+        TitusModelAction.Builder modelBuilder = TitusModelAction.newModelUpdate(changeActionBuilder).summary(summary);
         actions.add(ModelActionHolder.reference(modelBuilder.addTaskHolder(
                 EntityHolder.newRoot(newTask.getId(), newTask).addTag(TaskRetryers.ATTR_TASK_RETRY, newRetryer)
```

Comment:
`CallMetadata` should be added in the reference model, not the store model as below.